src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -75,12 +75,17 @@ private TranscriptionEntity saveTranscription(UserAccountEntity userAccount,
         }
 
         TranscriptionEntity transcription = new TranscriptionEntity();
-
+        HearingEntity hearing = null;
         if (!isNull(transcriptionRequestDetails.getHearingId())) {
```

Comment:
There are other methods in the Objects class like `nonNull` that could be used instead of `!isNull`, but I wonder why people use these methods outside of Predicates.  The JavaDoc says `This method exists to be used as a java.util.function.Predicate, filter(Objects::isNull)` and similarly for `nonNull`.
Prefer != null myself.

Reply:
In my opinion it looks neater.